frontend/mulighetsrommet-veileder-flate/src/components/modal/StatusModal.tsx

```diff
@@ -35,17 +34,15 @@ export function StatusModal({
 }: StatusModalProps) {
   const ikon = () => {
     if (ikonVariant === "success")
-      return (
-        <CheckmarkCircleFillIcon className={classNames(modalStyles.icon, svgStyle.svg_success)} />
-      );
+      return <CheckmarkCircleFillIcon className={classNames(modalStyles.icon, "text-green-600")} />;
     else if (ikonVariant === "warning")
       return (
         <ExclamationmarkTriangleFillIcon
-          className={classNames(modalStyles.icon, svgStyle.svg_warning)}
+          className={classNames(modalStyles.icon, "text-orange-600")}
         />
       );
     else if (ikonVariant === "error")
-      return <XMarkOctagonFillIcon className={classNames(modalStyles.icon, svgStyle.svg_error)} />;
+      return <XMarkOctagonFillIcon className={classNames(modalStyles.icon, "text-nav-red")} />;
   };
```

Comment:
Er denne `const ikon = `-komponenten basically den samme som den som er definert i `Feilmelding.tsx` over? Kunne muligens blitt trukket ut i en egen komponent som tok inn `variant` som en prop i stedet.

Reply:
Ja, jeg har ikke i hovedsak fokusert for mye på refaktorering i denne PRen. Tenkte å ta det i en egen PR (gå gjennom litt komponenter og se om vi kan generalisere litt mer). 

Jeg vet vi blant annet har en del forbedringspotensiale på en del komponenter som burde kunne generaliseres.